src/dbnode/storage/namespace.go

```diff
@@ -373,6 +373,7 @@ func (n *dbNamespace) SetSchemaHistory(value namespace.SchemaHistory) {
 	schema, ok := value.GetLatest()
 	if !ok {
 		n.log.Error("can not update namespace schema to empty", zap.Stringer("namespace", n.ID()))
+		return
```

Comment:
Should we add a test for this too? or is it covered by the other test you just added?

Reply:
yes, it is covered by the test